src/main/java/com/mraof/minestuck/data/loot_table/MSGiftLootTables.java

```diff
@@ -635,6 +639,7 @@ public void accept(BiConsumer<ResourceLocation, LootTable.Builder> lootProcessor
 						.addEntry(ItemLootEntry.builder(Items.FLOWER_POT).weight(5))
 						.addEntry(ItemLootEntry.builder(Items.BIRCH_BUTTON).weight(5))
 						.addEntry(ItemLootEntry.builder(MSItems.SORD).weight(10))
+						.addEntry(ItemLootEntry.builder(MSItems.CROWBAR).weight(3)).acceptFunction(SetDamage.func_215931_a(RandomValueRange.of(0.05F, 0.7F)))
```

Comment:
Parenthesis here is slightly wrong, and you ended up setting the loot function on the whole loot pool.
Should be `.addEntry(ItemLootEntry.builder(MSItems.CROWBAR).weight(3).acceptFunction(SetDamage.func_215931_a(RandomValueRange.of(0.05F, 0.7F))))`